docs/timezone.md

```diff
@@ -127,36 +127,59 @@ Effectively, this is the canonicalized version of whatever `timeZoneIdentifier`
 
 ## Methods
 
-### timeZone.**getOffsetFor**(_absolute_: Temporal.Absolute) : string
+### timeZone.**getOffsetFor**(_absolute_: Temporal.Absolute) : number
```

Comment:
Did we discuss whether this should return Number or BigInt?

Reply:
We didn't, but I put Number in the draft because the extreme values are ±86399999999999 (1 ns less than ±24:00) which is a safe integer.

Reply:
@pdunkel @littledan opinions on this?

Reply:
Temporal.DateTime's [[Nanoseconds]] slot is a Number, so it's fine for this one to also be a Number.  I'll resolve this thread.